desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -112,6 +113,7 @@ The following `admin-settings.json` code and table provides an example of the re
 | `configurationFileVersion`        |   |Specifies the version of the configuration file format.    |
 | `exposeDockerAPIOnTCP2375` | <span class="badge badge-info">Windows only</span>| Exposes the Docker API on a specified port. If `value` is set to true, the Docker API is exposed on port 2375. Note: This is unauthenticated and should only be enabled if protected by suitable firewall rules.| 
 | `proxy` |   |If `mode` is set to `system` instead of `manual`, Docker Desktop gets the proxy values from the system and ignores and values set for `http`, `https` and `exclude`. Change `mode` to `manual` to manually configure proxy servers. If the proxy port is custom, specify it in the `http` or `https` property, for example `"https": "http://myotherproxy.com:4321"`. The `exclude` property specifies a comma-separated list of hosts and domains to bypass the proxy. |
+&nbsp; &nbsp; &nbsp; &nbsp;`windowsDockerdPort`  | <span class="badge badge-info">Windows only</span> | Expose Docker Desktop's internal proxy locally on this port for the Windows Docker daemon to connect to. If it is set to 0, a random free port will be chosen. If the value is > 0, use that exact value for the port. The default value is -1 and it means that the option is disabled. |
```

Comment:
Can we add a note here that this option is only available for Windows containers?

Reply:
Good point!
Maybe change `Windows only` to `Windows containers only`?
Or  a `Note: This is available for Windows containers only.` at the beginning/end of the description?

Reply:
`Note: This is available for Windows containers only.` at the end of the description looks consistent with other entries here